web/tendrl/cluster_context_switcher.rst

```diff
@@ -0,0 +1,134 @@
+Context switcher
+****************
+
+:authors: 
+          - fbalak@redhat.com
+
+Description
+===========
+
+Change context of tendrl UI - show items in UI related to selected cluster.
+Context switcher is the dropdown menu rendered in masthead for each page
+except ``Clusters`` in UI.
+
+Setup
+=====
+
+#. Prepare multiple Gluster clusters (more than three) with created volumes.
+
+#. Login user into Tendrl, see :doc:`/web/tendrl/login_positive`
+
+#. Import all clusters, see :doc:`/web/tendrl/cluster_import`
+
+Test Steps
+==========
+
+.. test_action::
+   :step:
+       Check that there is no left panel shown after imports.
+   :result:
+       There is no left panel with menu visible on left side and context
+       switcher is set to ``All clusters``.
+
+.. test_action::
+   :step:
+       Click on the name of first imported cluster in the cluster list.
```

Comment:
Ah, so could you highlight that by "cluster list" you don't mean list of clusters in context switcher?